examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -0,0 +1,406 @@
+# Matter nRF Connect All Clusters Example Application
+
+The nRF All Clusters Example Application implements various ZCL clusters populated on three endpoints.
+You can use this example as a reference for creating your own application.
+
+<p align="center">
+  <img src="../../platform/nrfconnect/doc/images/Logo_RGB_H-small.png" alt="Nordic Semiconductor logo"/>
+  <img src="../../platform/nrfconnect/doc/images/nRF52840-DK-small.png" alt="nRF52840 DK">
+</p>
+
+The example is based on
+[Matter](https://github.com/project-chip/connectedhomeip) and Nordic
+Semiconductor's nRF Connect SDK, and was created to facilitate testing and certification of a Matter device communicating over a low-power, 802.15.4 Thread network.
+
+The example behaves as a Matter accessory, that is a device that can be paired
+into an existing Matter network and can be controlled by this network.
+
+<hr>
+
+-   [Overview](#overview)
+    -   [Bluetooth LE advertising](#bluetooth-le-advertising)
+    -   [Bluetooth LE rendezvous](#bluetooth-le-rendezvous)
+-   [Requirements](#requirements)
+    -   [Supported devices](#supported_devices)
+-   [Device UI](#device-ui)
+-   [Setting up the environment](#setting-up-the-environment)
+    -   [Using Docker container for setup](#using-docker-container-for-setup)
+    -   [Using native shell for setup](#using-native-shell-for-setup)
+-   [Building](#building)
+    -   [Removing build artifacts](#removing-build-artifacts)
+    -   [Building with release configuration](#building-with-release-configuration)
+    -   [Building with low-power configuration](#building-with-low-power-configuration)
```

Comment:
Plase add `-   [Building with Device Firmware Upgrade support](#building-with-device-firmware-upgrade-support)` to link the build with DFU section you added.